### ISSUE TITLE
On Windows, for ID, Forward, RRA, CMC, Analyze, the default output directory uses "\"s

### ISSUE THREAD

**Issue by carmichaelong** [opensim-org/opensim-gui#1063]
### Steps to reproduce
1. Open any model
2. Open ID, Forward, RRA, CMC, or Analyze tool
3. Look at the default name given for the output folder

### Expected result
A path with "/"s

### Actual result
A path with "\"s

### Environment and GUI version
Windows ONLY. Mac correctly has "/"s in path.

**Comment by aymanhab** [comment]
@carmichaelong @chrisdembia is this an issue because we try to use the same files cross platform? We use paths returned by file browsers as is but maybe we need to store in unix only form so that they are cross platform, or just use Mac 😄

**Comment by carmichaelong** [comment]
Exactly, we thought that this might be an issue depending on how they are saved in setup files so that they could be used cross platform. I haven't done any testing to specifically check how these paths are stored in files from a Windows machines yet, though.

**Comment by chrisdembia** [comment]
Fixing this would be fantastic, because it's very common for users to send around non-portable setup files. However, I think this can be addressed after the 4.0 release.
